modules/ROOT/pages/security/security.adoc

```diff
@@ -0,0 +1,68 @@
+= Security Aspects of Infinite Scale
+:toc: right
+
+:description: This document gives you an overview of the different security measures of Infinite Scale.
+
+== Introduction
+
+{description}
+
+== Operational aspects
+
+=== API Gateway
+
+Infinite Scale uses a microservices architecture. But instead of handling authorization and authentication in each service, we have implemented an API gateway (the xref:deployment/services/proxy.adoc[proxy service]) to handle these tasks. All requests have to go through this gateway. This has the nice effect that all other services can be hidden behind a firewall, which reduces the vulnerability to attacks dramatically.
```

Comment:
```suggestion
Infinite Scale uses a microservices architecture. But instead of handling authorization and authentication in each service, we have implemented an API gateway (the xref:deployment/services/proxy.adoc[proxy service]) to handle these tasks. All requests have to go through this gateway. This has the nice effect that all other services can be hidden behind a firewall, which reduces the attack surface dramatically.
```

I would keep attack surface here as this is a common term. https://en.wikipedia.org/wiki/Attack_surface